<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deletar</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Editar dados do Colaborador</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_colab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Colaborador:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_app1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicativo 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cargo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Setor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_app3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicativo 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_app2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicativo 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_app4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicativo 4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_app5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicativo 5:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	background-color: rgb(170, 0, 0);
}
QPushButton:hover{
	background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	background-color: rgb(0, 170, 0);
}
QPushButton:hover{
	
	background-color: rgb(0, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>Alterar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_cpf">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_email">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_cargo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_setor">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_app1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_app2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_app3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_app4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_app5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
